libs/designsystem/src/lib/components/list/list.component.integration.spec.ts

```diff
@@ -76,8 +76,8 @@ describe('ListComponent', () => {
     it('should render first and last item with correct padding', async () => {
       const firstItem = itemsInList[0].shadowRoot.querySelector('.item-native');
       const lastItem = itemsInList[itemsInList.length - 1].shadowRoot.querySelector('.item-native');
-      expect(firstItem).toHaveComputedStyle({ 'padding-top': size('xxs') });
-      expect(lastItem).toHaveComputedStyle({ 'padding-bottom': size('xxs') });
+      expect(firstItem).toHaveComputedStyle({ 'padding-top': '0px' });
+      expect(lastItem).toHaveComputedStyle({ 'padding-bottom': '0px' });
```

Comment:
Do you think these tests have any value now? If not we could remove them.

Reply:
That's a good point! Let's remove them :)